srcpkgs/perl-Params-ValidationCompiler/template

```diff
@@ -1,18 +1,18 @@
 # Template build file for 'perl-Params-ValidationCompiler'.
 pkgname=perl-Params-ValidationCompiler
-version=0.27
+version=0.30
 revision=1
 noarch=yes
 wrksrc="${pkgname/perl-/}-${version}"
 build_style=perl-module
 hostmakedepends="perl"
 makedepends="perl perl-Eval-Closure perl-Exception-Class"
 depends="${makedepends}"
-checkdepends="perl-Specio perl-Test2-Suite perl-Test-Without-Module perl-Test2-Plugin-NoWarnings
- perl-Term-Table"
+checkdepends="perl-Specio perl-Test2-Suite perl-Test-Without-Module
+ perl-Test2-Plugin-NoWarnings perl-Term-Table"
 short_desc="Build an optimized subroutine parameter validator once, use it forever"
 maintainer="newbluemoon <blaumolch@mailbox.org>"
 homepage="https://metacpan.org/release/Params-ValidationCompiler"
 license="Artistic-2.0, GPL-1"
```

Comment:
in [LICENSE](https://metacpan.org/source/DROLSKY/Params-ValidationCompiler-0.30/LICENSE) I don't see `GPL-1`

Reply:
At the bottom of https://metacpan.org/pod/Params::ValidationCompiler, COPYRIGHT AND LICENSE:

> The Artistic License 2.0 (GPL Compatible)

Question is what “GPL Compatible” means: GPL-1.0-only or GPL-1.0-or-later?

Reply:
Or should it only be “Artitstic-2.0”?

Reply:
I guess `"Artistic-2.0"` is enough...

Reply:
Neither am I! ;) But I think you’re right and removed GPL. :)